resources/examples/simple-wrapper.m4

```diff
@@ -10,16 +10,17 @@
 
 # [ <-- needed because of Argbash
 
-script="$script_dir/simple.sh"
-test -f "$script" || { echo "Missing the wrapped script, was expecting it next to me, in '$script_dir'."; exit 1; }
+script="${script_dir}/simple.sh"
+test -f "${script}" || {
```

Comment:
Please start the next line with the operator (and presumably the block start)

Reply:
I think there's a problem with that idea.

Take a bash script named `test.bash`:
```bash
#!/bin/bash 
echo "Contents of ${0}"
cat "${0}"
echo ""
echo "Script output:"
foo="no such file.txt"
test -f "${foo}"
|| { 
    echo "file not found"
}
```

Running bash test.bash will give you:
```
Contents of test.bash
#!/bin/bash 
echo "Contents of ${0}"
cat "${0}"
echo ""
echo "Script output:"
foo="no such file.txt"
test -f "${foo}"
|| { 
    echo "file not found"
}

Script output:
test.bash: line 8: syntax error near unexpected token `||'
test.bash: line 8: `|| { '
```
I can start the block on a new line, but that looks weird, to have a block sitting out in the middle and having to go up to understand whats going on.  Normally I see things like:
- `command` `operator` `{ block }`
- `{ block }` `redirection` `$(command)`

and if the block has multiple commands the `{` and `}` are on their own lines from the block commands so the block indentation is clear